### PR TITLE
Add sticky headers on homepage table

### LINKED ISSUE #146: Scrolling header for table
**Is your feature request related to a problem? Please describe.**
User is unable to know the context of the table data fields when the header goes out of the screen. 

**Describe the solution you'd like**
Something like this:

**Describe alternatives you've considered**
-

**Additional context**
-

### PR DESCRIPTION
**Pull request by bnap00**
Fixes #146

### CODE REVIEW

**Comment by jeremyphilemon** [comment]
Could you review this again, unable to reproduce it?

File: src/App.scss [hunk 1]
```diff
@@ -530,7 +530,13 @@ table {
     }
   }
 }
-
+.stickyTable {
+  thead {
+    position: sticky;
+    top: 0;
+    background-color: white;
```
Comment (jeremyphilemon) [inline src/App.scss]: Let's not change the background color :)

Comment (bnap00) [reply src/App.scss]: actually it makes the header translucent, and hinders visibility, 
check this out 👇 
Screenshot 2020-03-26 at 12 10 42 PM

**Comment by bnap00** [comment]
works for me 
Screenshot 2020-03-26 at 12 15 10 PM

**Comment by bnap00** [comment]
looks like chrome is not supporting CSS property `position: sticky`, 
but the [artical](https://developers.google.com/web/updates/2016/12/position-sticky) suggests that chrome should support it

**Comment by satishsingh35** [comment]
@bnap00 apply sticky to all 'th', to get 'sticky' to work. 'sticky' does not seem to work on table element 

@jeremyphilemon Can you please add me as contributor to this project?

**Comment by bnap00** [comment]
@satishsingh35 I think that did the trick

**Comment by shuklaayush** [comment]
Just a thought. Should we also change the font size and background of the header row to match the last row (total)?

**Comment by bnap00** [comment]
> Just a thought. Should we also change the font size and background of the header row to match the last row (total)?

the total row is also transparent. 
cannot do anything that is transparent 
Screenshot 2020-03-26 at 3 55 13 PM

**Comment by jeremyphilemon** [comment]
Wasn't able to test this out locally :/

But thank you, closing this because I went with a later submitted patch that worked!

**Pull request closed without merging**